Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -8538,7 +8537,8 @@ void WebPage::requestTextRecognition(Element& element, TextRecognitionOptions&&
 void WebPage::updateWithTextRecognitionResult(const TextRecognitionResult& result, const ElementContext& context, const FloatPoint& location, CompletionHandler<void(TextRecognitionUpdateResult)>&& completionHandler)
 {
     auto elementToUpdate = elementForContext(context);
```

Comment:
I could not absorb this into the dynamicDowncast as there's `nodeAtLocation->shadowHost() != elementToUpdate` below.